packages/blade/src/utils/makeAccessible/makeAccessible.d.ts

```diff
@@ -0,0 +1,278 @@
+import type { AccessibilityRole } from 'react-native';
+export { default } from './makeAccessible.web';
+
+// All the WAI-ARIA 1.1 role attribute values from https://www.w3.org/TR/wai-aria-1.1/#role_definitions
+type AriaRoles =
+  | AccessibilityRole
+  | 'alert'
+  | 'alertdialog'
+  | 'application'
+  | 'article'
+  | 'banner'
+  | 'button'
+  | 'cell'
+  | 'checkbox'
+  | 'columnheader'
+  | 'combobox'
+  | 'complementary'
+  | 'contentinfo'
+  | 'definition'
+  | 'dialog'
+  | 'directory'
+  | 'document'
+  | 'feed'
+  | 'figure'
+  | 'form'
+  | 'grid'
+  | 'gridcell'
+  | 'group'
+  | 'heading'
+  | 'img'
+  | 'link'
+  | 'list'
+  | 'listbox'
+  | 'listitem'
+  | 'log'
+  | 'main'
+  | 'marquee'
+  | 'math'
+  | 'menu'
+  | 'menubar'
+  | 'menuitem'
+  | 'menuitemcheckbox'
+  | 'menuitemradio'
+  | 'navigation'
+  | 'none'
+  | 'note'
+  | 'option'
+  | 'presentation'
+  | 'progressbar'
+  | 'radio'
+  | 'radiogroup'
+  | 'region'
+  | 'row'
+  | 'rowgroup'
+  | 'rowheader'
+  | 'scrollbar'
+  | 'search'
+  | 'searchbox'
+  | 'separator'
+  | 'slider'
+  | 'spinbutton'
+  | 'status'
+  | 'switch'
+  | 'tab'
+  | 'table'
+  | 'tablist'
+  | 'tabpanel'
+  | 'term'
+  | 'textbox'
+  | 'timer'
+  | 'toolbar'
+  | 'tooltip'
+  | 'tree'
+  | 'treegrid'
+  | 'treeitem'
+  | (string & Record<string, unknown>);
+
+export type AccessibilityProps = CommonAccessibilityProps;
+export type CommonAccessibilityKeys = keyof CommonAccessibilityProps;
+export type AccessibilityMap = Record<CommonAccessibilityKeys, string>;
+
+export type CommonAccessibilityProps = {
+  role: AriaRoles;
+  /** Identifies the currently active element when DOM focus is on a composite widget, textbox, group, or application. */
```

Comment:
nit(can ignore): let's make these consistent?
```suggestion
  /**
  * Identifies the currently active element when DOM focus is on a composite widget, textbox, group, or application.
  */
```